.gitlab-ci.yml

```diff
@@ -1185,6 +1187,20 @@ deploy_puppy:
     - export PACKAGE_VERSION=$(inv agent.version --url-safe)
     - aws s3 cp --region us-east-1 ./agent $S3_DSD6_URI/puppy/agent-$PACKAGE_VERSION --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
 
+# deploy process agent and system-probe to staging bucket
+deploy_process_and_sysprobe:
+  stage: internal_deploy
+  when: manual
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/deploy:latest
+  before_script:
+    - cd $OMNIBUS_PACKAGE_DIR
```

Comment:
Does the cd propagate to the next comane? Elsewhere is: `    - ls $OMNIBUS_PACKAGE_DIR` (all one command)

https://github.com/DataDog/datadog-agent/pull/3559/files#diff-96edf7a6f008de9e928d04e1ae5e12a5R1183

Reply:
It does for the rest of the script 👍